### ISSUE TITLE
Web UI default auth backend is other

### ISSUE THREAD

**Issue by yura-shutkin** [hashicorp/vault#7949]
**Describe the bug**
Since 1.3.0, if you have listed backends for unauthorized users, by default selected auth backend is other

![Screenshot from 2019-11-28 10-16-38](https://user-images.githubusercontent.com/8489762/69779087-9b9b5080-11c8-11ea-805a-ce1c53014d19.png)

Before 1.3.0 it was ldap, userpass, github, etc.

![Screenshot from 2019-11-28 10-19-12](https://user-images.githubusercontent.com/8489762/69779094-a524b880-11c8-11ea-94dd-983ab4fb1b42.png)

**To Reproduce**
1. Run `docker run --rm -d -p 0.0.0.0:8200:8200 --name=vault-public -e VAULT_DEV_ROOT_TOKEN_ID=12345 vault:1.3.0`
2. Run `export VAULT_ADDR=http://localhost:8200`
3. Run `vault login 12345`
4. Run `vault auth enable -listing-visibility=unauth ldap`
5. Go to http://localhost:8200 and see that selected auth method is other

**Expected behavior**
By default should be selected auth backend with listing_visibility equals unauth

**Environment:**
* Vault Server Version (retrieve with `vault status`):
```
Key             Value
---             -----
Seal Type       shamir
Initialized     true
Sealed          false
Total Shares    1
Threshold       1
Version         1.3.0
Cluster Name    vault-cluster-7218622a
Cluster ID      1a033508-8cfa-808e-4f5c-2da01f1749b9
HA Enabled      false
```
* Vault CLI Version (retrieve with `vault version`):
```
Vault v1.3.0
```
* Server Operating System/Architecture:
```
Linux dell 5.0.0-36-generic #39~18.04.1-Ubuntu SMP Tue Nov 12 11:09:50 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
ubuntu 18.04
```

Vault server configuration file(s):

even default dev configuration

**Additional context**
It is not a major bug, but I can't use 1.3.0 in production cluster be cause of it. I'll got a lot of messages  like: "LDAP auth backend is not working !" I already got messages like this on testing env

**Comment by yura-shutkin** [comment]
Found a mistake in vault version. Should be 1.3.0
```
Key             Value
---             -----
Seal Type       shamir
Initialized     true
Sealed          false
Total Shares    1
Threshold       1
Version         1.3.0
Cluster Name    vault-cluster-7218622a
Cluster ID      1a033508-8cfa-808e-4f5c-2da01f1749b9
HA Enabled      false
```

**Comment by Mrono** [comment]
I'm unable to upgrade to 1.3.x as well due to this

**Comment by Monkeychip** [comment]
@yura-shutkin thank you for the issue.  This will be fixed in version 1.3.3.
